web/src/components/WelcomeModal.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalBody,
+    ModalFooter,
+    Button,
+  } from "@chakra-ui/react";
+  import { useState } from "react";
+  import LAPTOP_CIRCLE_ICON from "../assets/laptop-circle-icon.svg";
+  
+  export function WelcomeModal() {
+    const [openModal, setOpenModal] = useState(true);
+  
+    function handleCloseModal() {
+      setOpenModal(false);
```

Comment:
It's going to be annoying if this opens every single time. I still disagree with the decision to put a popup on the demo. Let's make sure that the user doesn't see it multiple times by setting something in local storage. Create an atom in recoil js that is attached to a local storage effect, and set it here. Then only show the modal if it is not set.